delivery_package_number/wizard/stock_backorder_confirmation.py

```diff
@@ -15,9 +15,14 @@ class StockBackorderConfirmation(models.TransientModel):
     @api.depends("pick_ids")
     def _compute_ask_number_of_packages(self):
         for item in self:
-            item.ask_number_of_packages = bool(item.pick_ids.carrier_id)
+            item.ask_number_of_packages = bool(
+                item.pick_ids.carrier_id and not item.pick_ids._origin.package_ids
```

Comment:
Why this?

Reply:
It's incorrect to use `_origin` in a compute, as it's using what was stored last time, which is nothing that can be compared now in the current record state.

Reply:
I use `_origin` because otherwise it does not get the `picking_ids`. 

That is, if I use `item.picking_ids` it returns `stock.picking(<NewId origin=XX>,)` and `item.picking_ids.package_ids` always returns `stock.quant.package()` even if the `stock.picking` does have `package_ids`.

How else can I do it?

Reply:
OK, being a transient model which is populated only for this and no more changes are done over `picking_ids`, it's acceptable, but please put a code comment explaining why that "design decission" for your future you and for the rest to know why.

Reply:
I have put a comment in the code, what do you think?
THX